packages/frontend-2/pages/index.vue

```diff
@@ -1,212 +1,22 @@
-                        name="Disabled"
-                        placeholder="This field is disabled"
-                        disabled
-                      />
-                    </div>
-                    <div>
-                      <FormTextInput
-                        name="Email"
-                        placeholder="test@text.com"
-                        type="email"
-                      />
-                    </div>
-                    <div>
-                      <FormTextInput
-                        name="Password"
-                        placeholder="Enter PW here"
-                        type="password"
-                      />
-                    </div>
-                    <div>
-                      <FormCheckbox name="Checkbox1" />
-                    </div>
-                    <div>
-                      <FormCheckbox
-                        name="Checkboxreq"
-                        label="Required checkbox"
-                        show-required
-                        :rules="(val) => (val ? true : 'Required!')"
-                      />
-                    </div>
-                    <div>
-                      <FormCheckbox
-                        name="Checkbox2"
-                        label="Custom label"
-                        description="Here's a description!"
-                      />
-                    </div>
-                    <div>
-                      <FormCheckbox
-                        name="Checkbox3"
-                        description="Here's an inline description!"
-                        inline-description
-                      />
-                    </div>
-                    <div>
-                      <FormCheckbox
-                        name="errorcheckbox1"
-                        label="Checkbox with error"
-                        description="Hello world!"
-                        :rules="
-                          (val) =>
-                            val ? true : 'Check this in to get rid of the error!'
-                        "
-                        validate-on-mount
-                      />
-                    </div>
-                    <div><FormButton submit full-width>Submit</FormButton></div>
-                  </Form>
-                </div>
-                <div>And that's all, folks!</div>
-              </div>
-            </template>
-            <template #footer>
-              <FormButton to="/">Example panel footer button</FormButton>
-            </template>
-          </LayoutPanel>
-        </div>
-      </div>
+    <!-- <header class="default-width">
+      <section>
+        <h1 class="h3 text-foreground">Henlo 👋</h1>
+        <p class="text-foreground">
+          Welcome {{ user?.name }}! This is a super big work in progress currently.
+          <NuxtLink class="link" to="/designsystem">Design System Demo</NuxtLink>
+        </p>
+      </section>
+    </header> -->
+    <main class="default-width">
```

Comment:
IMO the semantic HTML5 `<main>` element should be used somewhere at the layout level, cause otherwise devs will forget to add it in all pages, making the semantics of the page inconsistent